src/middleware/cors/corsConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { CorsOptions } from 'cors';
+
+// List of allowed origins
+const allowedOrigins: string[] = [
```

Comment:
Can we put this in the env files? The full array if needed, or depending on the environment it should have according vlaue.

Reply:
nice Idea, I thought about that :) will change

Reply:
Done